src/views/product-downloads-view/index.tsx

```diff
@@ -75,23 +71,6 @@ const ProductDownloadsViewContent = ({
 			isEnterpriseMode
 		),
 	]
-	const packageManagers = useMemo(() => {
```

Comment:
We build `packageManagers` in `getStaticProps`, so this client-side work is no longer needed.